make.bat

```diff
@@ -68,10 +70,10 @@ cd ..
 set include=x86\include\
 CALL:start_timer
 ECHO === Building Linux Executables ===
-start /min /wait fasmg.exe "x86\fish.asm" "asmFishL_%datestamp%_popcnt" -e 1000 -i "VERSION_OS='L'" -i "PEDANTIC = 1" -i "VERSION_POST = 'popcnt'" %debug%
```

Comment:
This is just a safety precaution to not overwhelm weak computers with too many instructions. It doesn't hurt anything here.

Reply:
@Counterply, weak computers use base executables, correct? So, I should put start /min /wait when assembling base executables, correct?